.buildkite/pipeline.yml

```diff
@@ -11,6 +11,8 @@ steps:
           - BUILDKITE_BUILD_NUMBER
           - BUILDKITE_BRANCH
           - AWS_LOGS_BUCKET
+    agents:
+      queue: 't2medium'
```

Comment:
The unit tests for the Ensemble class require more memory that any other test since we make copies of models. The small instance with 1GB of memory that was running our tests would basically run out of memory. I talked with @TheDen and with some testing we decided that we need a larger instance to run the tests. I've tried to reduce the amount of RAM needed but unfortunately, it was still at the limit and the largest instance is the way to go.